python/mxnet/ndarray/sparse.py

```diff
@@ -88,6 +88,8 @@ def _new_alloc_handle(stype, shape, ctx, delay_alloc, dtype, aux_types, aux_shap
         A new empty ndarray handle
     """
     hdl = NDArrayHandle()
+    for aux_t in aux_types:
+        assert(np.dtype(aux_t) == np.dtype("int64")), "only int64 is supported for aux types"
```

Comment:
Is using `raise` more appropriate here? In production, users may not want to terminate their program.

Reply:
raising UnimplementedError now